python_modules/dagster/dagster/core/asset_defs/decorators.py

```diff
@@ -301,6 +303,131 @@ def inner(fn: Callable[..., Any]) -> AssetsDefinition:
     return inner
 
 
+@experimental_decorator
+def assets_definition(
+    name: Optional[str] = None,
+    asset_key_by_input_name: Optional[Mapping[str, AssetKey]] = None,
+    asset_key_by_output_name: Optional[Mapping[str, AssetKey]] = None,
+    internal_asset_deps: Optional[Mapping[str, Set[AssetKey]]] = None,
+) -> Callable[[Callable[..., Any]], AssetsDefinition]:
+    """Create a combined definition of multiple assets that are computed using the same op and same
```

Comment:
Nitpick, but I might change the wording here a bit to something like "Create a definition of multiple assets which share the same computation and same upstream assets", just to make it a little more future-proof (for when graphs can be annotated as well)